### DISCUSSION TITLE
"You are using an anti-detect browser" when using mobile mode

### DISCUSSION THREAD

**Discussion by longnguyenQB** [seleniumbase/SeleniumBase#2246] [Q&A] [answered]
Hello! I have problems with mobile mode when accessing pixelscan.net. There it says "You are using an anti-detect browser"
Here is simple code:
''
from seleniumbase import Driver
import time

driver = Driver( uc=True, mobile= True, is_mobile= True)
try:
    driver.default_get("https://pixelscan.net/")
    time.sleep(10000)
finally:
    driver.quit()
''

**Comment by mdmintz** [comment, marked as answer]
`driver.default_get()` should only be used when you **don't** want to open a website using UC Mode. It **disables** UC Mode for that website, as mentioned in https://www.youtube.com/watch?v=5dMFI3e85ig.

Use these for special UC Mode URL opens:

```python
driver.uc_open(url)
driver.uc_open_with_tab(url)
driver.uc_open_with_reconnect(url, reconnect_time=None)
```

Use this for a UC Mode click:

```python
driver.uc_click(selector)
```

UC Mode works with Mobile Mode. (See https://github.com/seleniumbase/SeleniumBase/issues/2169 for details and video.)

**Reply by longnguyenQB** [reply]
Great! Thanks for your work! uc mode can combine with mobile mode, they have surpassed "https://nowsecure.nl/#relax", however for "https://pixelscan.net/" I still get the message "You are using an anti-detect browser". I'm trying to use mobile mode to work on tiktok but it seems they still detect my bot.. Here is my code. Hope I'm wrong 

```
from seleniumbase import SB

with SB(uc=True, mobile=True) as sb:
    sb.driver.uc_open("https://pixelscan.net/")
```

**Reply by mdmintz** [reply]
You can hide that you're using an automation framework: `No automation framework detected`.
But you can't fake using a real mobile device when you're not actually using a real one.

```python
from seleniumbase import SB

with SB(uc=True, mobile=True) as sb:
    sb.driver.uc_open_with_reconnect("https://pixelscan.net/", 15)
    import pdb; pdb.set_trace()
```

<img width="680" alt="Screenshot 2023-11-07 at 1 31 27 PM" src="https://github.com/seleniumbase/SeleniumBase/assets/6788579/95451907-8af5-4af5-bdb1-2e76d82d8020">

**Reply by longnguyenQB** [reply]
Thank you very much.  So is there any way to run undetected chrome on a real mobile?  It would be great if we had undetected on pc and also on mobile 😄

**Reply by mdmintz** [reply]
That would have to be a separate project by someone more familiar with https://appium.io/
Appium is a very different project from regular Selenium, but it's specially made for mobile.

**Reply by longnguyenQB** [reply]
Thank you very much for replying me so quickly. Wishing you all the best!
